charts/adot-exporter-for-eks-on-ec2/values.yaml

```diff
@@ -331,11 +337,58 @@ adotCollector:
       - dimensions: [[Namespace, ClusterName], [ClusterName]]
         metric_name_selectors:
           - namespace_number_of_running_pods
+    ampreceivers:
+      scrapeInterval: 15s
+      scrapeTimeout: 10s
+      scrapeConfigs: |
+            - job_name: 'storefront'
```

Comment:
This job name isn't correct, is it?

Reply:
Yeah, maybe not. I changed it to 'k8s_metrics_scrape' since the job also handles the Prometheus scraping configs.